src/github.com/getlantern/golog/golog_test.go

```diff
@@ -6,26 +6,34 @@ import (
 	"io/ioutil"
 	"os"
 	"regexp"
+	"strings"
 	"testing"
 	"time"
 
-	"github.com/getlantern/testify/assert"
```

Comment:
Yeah exactly @xiam. Ours was actually worse at displaying more standard test output, though, which is what we need for this particular test, so I reverted. I agree we should probably just remove getlantern/testify, as I'm also guessing similar functionality is now upstream.